contrib/opencensus-ext-pymongo/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased
+- Changed attributes names to match [specs](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md)
```

Comment:
We shouldn't link to the OT spec here.

Reply:
I have slightly different opinion on this.
I agree that OpenCensus Python should follow OpenCensus spec. Following OpenTelemetry spec is not the goal here, but it is nice to have if:
1. It doesn't go against OpenCensus spec.
2. It doesn't introduce major breaking changes (given we're moving to OpenTelemetry soon) without a good rationale.
3. It helps us to form better understanding on our path to OpenTelemetry, or simply make the job easier when we try to steal some code and morph them into OpenTelemetry.

With these, I think having a link to OT spec is similar like saying "we are supporting W3C CorrelationContext version 2" - not something required in OC spec, but a nice to have feature.

Reply:
I agree with all that, I'm just concerned about putting a link to the OT spec in something as public as the release notes. There's already a lot of confusion around OC/OT, linking the OT spec here might invite more.